kubetest/e2e.go

```diff
@@ -164,7 +164,14 @@ func run(deploy deployer, o options) error {
 			errs = appendError(errs, err)
 		} else {
 			errs = appendError(errs, xmlWrap("UpgradeTest", func() error {
-				return skewTest(argFields(o.upgradeArgs, dump, o.clusterIPRange), "upgrade", o.checkSkew)
+				// upgrade tests really only run one spec
+				var env []string
+				for _, v := range os.Environ() {
+					if !strings.HasPrefix(v, "GINKGO_PARALLEL") {
```

Comment:
you probably also want to handle `GINKGO_PARALLEL_NODES`

Reply:
and ideally we want to muck with ginkgo flags rather than global envs only useful for hack/ginkgo-e2e.sh